ibc-clients/ics07-tendermint/src/client_state/validation.rs

```diff
@@ -13,20 +13,24 @@ use ibc_primitives::proto::Any;
 use tendermint::crypto::default::Sha256;
 use tendermint::crypto::Sha256 as Sha256Trait;
 use tendermint::merkle::MerkleHash;
+use tendermint_light_client_verifier::{ProdVerifier, Verifier};
 
 use super::{check_for_misbehaviour_on_misbehavior, check_for_misbehaviour_on_update, ClientState};
 use crate::client_state::{verify_header, verify_misbehaviour};
-use crate::verifier::{DefaultVerifier, TmVerifier};
 
 impl<V> ClientStateValidation<V> for ClientState
 where
     V: ExtClientValidationContext,
     V::ConsensusStateRef: Convertible<ConsensusStateType, ClientError>,
 {
     /// The default verification logic exposed by ibc-rs simply delegates to a
-    /// standalone `verify_client_message` function. This is to make it as simple
-    /// as possible for those who merely need the `DefaultVerifier` behaviour, as
-    /// well as those who require custom verification logic.
+    /// standalone `verify_client_message` function. This is to make it as
+    /// simple as possible for those who merely need the default `ProdVerifier`
+    /// behaviour, as well as those who require custom verification logic.
+    ///
+    /// In situations when the Tendermint [`ProdVerifier`] doesn't provide the
+    /// desired outcome, users should define a custom verifier struct and then
+    /// implement the `tendermint_light_client_verifier::Verifier` trait for it.
```

Comment:
I think we can do this as `Verifier` is in the scope.

```suggestion
    /// implement the [`Verifier`] trait for it.
```